content/updates/2021-05/4-Product-completeness-in-the-API.md

```diff
@@ -0,0 +1,37 @@
+---
+pim_announcement_audience:
+- EE
+- CE
+---
+
+# Highlight the most enriched products on your channels
```

Comment:
```suggestion
# New API filter on product completeness
```
Suggesting a new title, to be more straight to the point, so people directly know what's new.